docs/docusaurus/docs/core/configure_project_settings/configure_data_docs/_examples/data_docs_abs.py

```diff
@@ -0,0 +1,100 @@
+"""
+This is an example script for how to configure Data Docs in an Azure Blob Storage environment.
+
+To test, run:
+
+"""
+
+
+def set_up_context_for_example(context):
+    # Create a Batch Definition
+    batch_definition = (
+        context.data_sources.add_pandas_filesystem(
+            name="my_data_source", base_directory="./data/folder_with_data"
+        )
+        .add_csv_asset(name="my_data_asset")
+        .add_batch_definition_path(
+            name="my_batch_definition", path="yellow_tripdata_sample_2019-01.csv"
+        )
+    )
+
+    # Create an Expectation Suite
+    expectation_suite = context.suites.add(
+        gx.ExpectationSuite(name="my_expectation_suite")
+    )
+    # Add some Expectations
+    expectation_suite.add_expectation(
+        gx.expectations.ExpectColumnValuesToNotBeNull(column="pickup_datetime")
+    )
+    expectation_suite.add_expectation(
+        gx.expectations.ExpectColumnValuesToNotBeNull(column="passenger_count")
+    )
+
+    # Create a Validation Definition
+    context.validation_definitions.add(
+        gx.ValidationDefinition(
+            data=batch_definition,
+            suite=expectation_suite,
+            name="my_validation_definition",
+        )
+    )
+
+
+# EXAMPLE SCRIPT STARTS HERE:
+# <snippet name="docs/docusaurus/docs/core/configure_project_settings/configure_data_docs/_examples/data_docs_abs.py - full code example">
+import great_expectations as gx
+
+context = gx.get_context(mode="file")
+# Hide this
+set_up_context_for_example(context)
+
+# Start a Data Docs configuration dictionary
+site_name = "my_data_docs_site"
+site_config = {
+    "site_name": site_name,
+    "class_name": "SiteBuilder",
+    "site_index_builder": {"class_name": "DefaultSiteIndexBuilder"},
+}
+
+# Add a Store backend configuration to the Data Docs configuration
+# <snippet name="docs/docusaurus/docs/core/configure_project_settings/configure_data_docs/_examples/data_docs_abs.py - add store backend">
+container = "my_abs_container"
+prefix = "data_docs/"
+connection_string = "${AZURE_STORAGE_CONNECTION_STRING}"  # This uses string substitution to get the actual connection string from an environment variable or config file.
+
+site_config["store_backend"] = {
```

Comment:
Tyler's preference alert: I find it way harder to reason about when we mutate objects, e.g. site_config is only partially built above and then we add another key to it here. I want one place to grok the shape of it.

Reply:
That's fair.  I originally split it because the store_backend is the only part that differs between environments, but I can merge the two sections back together easily enough now that I have sample scripts for each environment to pull snippets from.